openmdao/core/driver.py

```diff
@@ -48,6 +48,8 @@ def __init__(self):
         self.root = None
 
         self.iter_count = 0
+        self.dv_conversions = {}
```

Comment:
I would move the initialization of these into _setup so that if Problem.setup() gets called multiple times they will be initialized properly.